js/yarn.lock

```diff
@@ -2,6 +2,11 @@
 # yarn lockfile v1
 
 
+"@akiroz/size-chunker-stream@^0.0.1":
+  version "0.0.1"
```

Comment:
Why are you changing versions at this level?

Reply:
This & the `.yarnrc/.yarn` was the result of adding the `arbundles` dependency using yarn 0.18.0 cuz I didn't know about `lerna add ...`.

The diff in `yarn.lock` now only reflects changes made by `npx lerna add arbundles packages/cli`